build.gradle

```diff
@@ -127,7 +127,7 @@ jacocoTestReport {
     }
 }
 
-String version = '6.6.1'
+String version = '6.6.2'
```

Comment:
Should be at least a minor bump.

Reply:
Sure @peterzhuamazon, adding @gaiksaya for some inputs on the minor bump.

Reply:
Looking back again I agree since we added two new libs we should go with `6.7.0`.